angular/projects/spark-angular/src/lib/components/sprk-card/sprk-card.stories.ts

```diff
@@ -157,41 +338,289 @@ export const teaser = () => ({
  `,
 });
 
-teaser.story = {
-  name: 'Teaser',
+legacyTeaser.story = {
+  name: 'Legacy Teaser (Deprecated)',
   parameters: {
     docs: { iframeHeight: 550 },
     jest: ['sprk-card.component'],
   },
 };
 
+export const teaserIcon = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-card idString="teaser-icon">
+      <sprk-stack
+        sprkCardContent
+        itemSpacing="large"
+        sprkStackItem
+        additionalClasses="sprk-u-TextAlign--center"
+      >
+        <a
```

Comment:
There is an accessibility violation associated with this element.
![Screen Shot 2021-03-04 at 9 45 42 AM](https://user-images.githubusercontent.com/15703773/109980902-64bac100-7cce-11eb-9fa3-f8724d2a3f30.png)

Reply:
@guern1kn yeah looks like its there in PROD right now too for HTML and React. Yikes. https://html.sparkdesignsystem.com/?path=/story/components-card--three-up-cards -- I'll fix them all for us.

Reply:
added html, will add react with react work